src/azure-cli/azure/cli/command_modules/config/tests/latest/test_config.py

```diff
@@ -24,6 +24,7 @@ def test_config(self):
 
         # C:\Users\{username}\AppData\Local\Temp
         tempdir = tempfile.gettempdir()
+        tempdir = os.path.realpath(tempdir)
```

Comment:
Let's add a comment for this special case on MacOS.

Reply:
updated.